packages/apps/explorer/src/graphql/subscriptions/newBlocks.graph.ts

```diff
@@ -0,0 +1,17 @@
+import { gql } from '@apollo/client';
+import type { DocumentNode } from 'graphql';
+
+export const newBlocksSubscription: DocumentNode = gql`
```

Comment:
```suggestion
export const newBlocks: DocumentNode = gql`
```

The generated code will already postfix these with Query and Subscription